WordPress/Classes/Services/MediaImportService.swift

```diff
@@ -120,36 +343,24 @@ open class MediaImportService: LocalCoreDataService {
         DDLogError("\(errorLogMessage), code: \(nerror.code), error: \(nerror)")
     }
 
-    /// Handle the OnError callback and logging any errors encountered.
-    ///
-    fileprivate func handleExportError(_ error: MediaExportError, errorHandler: OnError?) {
-        MediaImportService.logExportError(error)
-        // Return the error via the context's queue, and as an NSError to ensure it carries over the right code/message.
-        if let errorHandler = errorHandler {
-            self.managedObjectContext.perform {
-                errorHandler(error)
-            }
-        }
-    }
-
     // MARK: - Media export configurations
 
-    fileprivate var exporterImageOptions: MediaImageExporter.Options {
+    private var exporterImageOptions: MediaImageExporter.Options {
         var options = MediaImageExporter.Options()
         options.maximumImageSize = self.exporterMaximumImageSize()
         options.stripsGeoLocationIfNeeded = MediaSettings().removeLocationSetting
         options.imageCompressionQuality = MediaImportService.preferredImageCompressionQuality
         return options
     }
 
-    fileprivate var exporterVideoOptions: MediaVideoExporter.Options {
+    private var exporterVideoOptions: MediaVideoExporter.Options {
         var options = MediaVideoExporter.Options()
         options.stripsGeoLocationIfNeeded = MediaSettings().removeLocationSetting
         options.exportPreset = MediaSettings().maxVideoSizeSetting.videoPreset
         return options
     }
```

Comment:
Nitpick. This doesn't access anything from `self`. Could it become a constant so we don't need to compute it on every access?